### ISSUE TITLE
210826 멘토님 질문 리스트

### ISSUE THREAD

**Issue by KimDaeUng** [boostcampaitech2/image-classification-level1-04#14]
# 210826 멘토님 질문 리스트
---

## Data
- [x] 육안상으로 보이는 이미지가 납득이 가지 않아도 Augmentation 기법을 쓸 수 있는지?
- [x] Normalize 시에 Mean, Std 값을 데이터 기준으로 주는 건지 pretrained model 기준으로 주는 건지?
## Model
- [x] pretrianed model 사용시 fine-tuning 적용 여부 판단
- [x] 적절한 데이터 로딩 방법?

## Etc.
- [x] 파이썬에서 모듈 임포트시 경로 설정이 헷갈립니다. 
- [x] GPU memory가 실험이 끝나도 계속 남아있는 경우, 어떻게 리셋하나요?

## 실험관리
- [x] Weghts & Biases 실험 관리를 팀 단위로 공유할 수 있나요?
